site/src/Librecores/ProjectRepoBundle/Entity/Project.php

```diff
@@ -126,7 +129,8 @@ class Project
      *
      * @var string
      *
-     * @ORM\Column(type="text", nullable=true)
+     * @Assert\Length(max = 50000)
```

Comment:
I agree with the size. (it's almost perfect - 64KB is the perfect one for obvious reasons!)